### ISSUE TITLE
Connected Method Missing from Templates

### ISSUE THREAD

**Issue by CBaker007** [ASCOMInitiative/AlpycaDevice#8]
I am a novice programmer and believe based on your videos that the goal is to make this package a very skinny and simple template for someone to get started writing an Alpaca Driver.  I really appreciate this package!!  I looked all over for days on how to get started writing my own ASCOM COM or Alpaca driver when I could not find something existing to do what I needed.   So I am not sure if this is an Issue or intentional.  **So please forgive me if it was intentional and feel free to ignore this post if it was.**

In the original example, there is a "connected" class in the rotator.py example (with both on_get and on_put).  From what I understand, all these classes in the <devicetype>.py (rotator.py for example) are used to generate the URI's (ASCOM Alpaca Methods) for each device.  From reading the documentation and the ASCOM API website, this "method" appears to be required of all devices (https://ascom-standards.org/api/#/ASCOM%20Methods%20Common%20To%20All%20Devices).  The example compiles and passes the ConformU  tests.  It appears that all the common and device specific methods except "connected" are in the templates.

But when trying to follow the directions for building your own driver using the templates, after trying to run ConformU, I kept getting "404 (NotFound)" errors on all "GET Connected" tests.  It took me a lot of time to finally figure out there is no "class connected" in the template I was using (and appears all).  Unless I missed it, the documentation did not mention this missing "class connected" in the template.   Without the "class connected", it fails conformity testing at the most basic level (404 errors).

So I added back a generic version of the rotator.py example "class connected:" as follows:

```
@before(PreProcessRequest(maxdev))
class connected:
    """Retrieves or sets the connected state of the device

    * Set True to connect to the device hardware. Set False to disconnect
      from the device hardware. Client can also read the property to check
      whether it is connected. This reports the current hardware state.
    * Multiple calls setting Connected to true or false must not cause
      an error.
   """

    def on_get(self, req: Request, resp: Response, devnum: int):
        resp.text = PropertyResponse(True, req).json

    def on_put(self, req: Request, resp: Response, devnum: int):
        try:
            resp.text = MethodResponse(req).json
        except Exception as ex:
            resp.text = MethodResponse(req, # Put is actually like a method :-(
                            DriverException(0x500, f'{self.__class__.__name__} failed', ex)).json
```

It still does not pass the ConformU due to some tests of "bad parameters" being passed (Empty string, Number, Meaningless string), but all of the 404 errors went away. 

I know this needs to actually test the connection status (on_get) and connect/disconnect (on_put) and to test the passed values are in range, but it gets rid of the 404 errors and let's me know I need to deal with this method.  I would recommend that either the a "class connected" be included with each template or the documentation notes the need for this class to be added to every template.

**Comment by BobDenny** [comment]
Yeah this is an ugly one! Will be fixed on 0.4.

**Comment by BobDenny** [comment]
Released 0.4 with this fixed in the templates.
